### PR TITLE
Update Packaging Pin to >=23.0

### PR DESCRIPTION
**Pull request by rxm7706**
## Description

resolves https://github.com/kedro-org/kedro-viz/issues/1946

## Checklist

- [X] Read the [contributing](/CONTRIBUTING.md) guidelines
- [X] Opened this PR as a 'Draft Pull Request' if it is work-in-progress
- [ ] Updated the documentation to reflect the code changes
- [ ] Added new entries to the `RELEASE.md` file
- [ ] Added tests to cover my changes

### CODE REVIEW

**Comment by astrojuanlu** [comment]
cc @ravi-kumar-pilla

**Comment by ravi-kumar-pilla** [comment]
Hi @rxm7706 , Thank you for the PR. Could you please add more information on #1946 . 

Details like -
1. How this bug affects your application
2. How can we reproduce this bug on our end
3. Your application environment (Kedro-Viz and python version, OS and version)
4. Any other details you feel would help us in debugging

These details would help us to understand the root cause and fix any future issues. Thank you !

File: RELEASE.md [hunk 1]
```diff
@@ -12,6 +12,7 @@ Please follow the established format:
 - Enable/disable preview for all the datasets when publishing Kedro-Viz from UI. (#1895)
 
 ## Bug fixes and other changes
+- Relax `Packaging` pin in requirements. (#1947)
```
Comment (deepyaman) [inline RELEASE.md:15]:
```suggestion
- Relax `packaging` pin in requirements. (#1947)
```
Nit :)

**Review by deepyaman — APPROVED** [review]
Very minor comment; otherwise, sounds good to me, to be able to get latest `packaging`!

**Comment by astrojuanlu** [comment]
hey @rxm7706, do you still intend to pursue this?

**Comment by rxm7706** [comment]
> hey @rxm7706, do you still intend to pursue this?

@astrojuanlu - Yes, please. . I will open another PR to address.

**Comment by rxm7706** [comment]
> > hey @rxm7706, do you still intend to pursue this?
> 
> @astrojuanlu - Yes, please. . I will open another PR to address.

continued - https://github.com/kedro-org/kedro-viz/pull/1953

**Pull request closed without merging**